modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -105,21 +103,15 @@ def valid_poa_code_for_current_user?(poa_code) # rubocop:disable Metrics/MethodL
       def verify_power_of_attorney!
         return if token.client_credentials_token?
 
-        logged_in_representative_user = @current_user
-        target_veteran_to_be_verified = target_veteran
-        verify_representative_and_veteran(logged_in_representative_user, target_veteran_to_be_verified)
+        target_veteran_to_verify = ::Veteran::User.new(target_veteran)
+        poa_code_to_verify = target_veteran_to_verify.power_of_attorney.try(:code)
+        valid_poa_code_for_current_user?(poa_code_to_verify)
```

Comment:
cf. https://github.com/department-of-veterans-affairs/vets-api/blob/3dac695f63ef2aadadb5c702c18daf0c00e909c0/lib/bgs/power_of_attorney_verifier.rb#L4-L16